bulldozer/merge.go

```diff
@@ -183,104 +183,128 @@ func MergePR(ctx context.Context, pullCtx pull.Context, merger Merger, mergeConf
 
 		message, err := calculateCommitMessage(ctx, pullCtx, *opt)
 		if err != nil {
-			return errors.Wrap(err, "failed to calculate commit message")
+			logger.Error().Err(err).Msg("Failed to calculate commit message")
+			return
 		}
 		commitMsg.Message = message
 
 		title, err := calculateCommitTitle(ctx, pullCtx, *opt)
 		if err != nil {
-			return errors.Wrap(err, "failed to calculate commit title")
+			logger.Error().Err(err).Msg("Failed to calculate commit title")
+			return
 		}
 		commitMsg.Title = title
 	}
 
-	go func(ctx context.Context) {
-		ticker := time.NewTicker(4 * time.Second)
```

Comment:
I removed the ticker because it doesn't write the first message to the channel until after `duration`, so I think we had an unnecessary 4s delay here.